### ISSUE TITLE
[Beta Tester Feedback] Order Drafts by post date instead of last edit

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-iOS#17362]
We received the following email (ref: Oct 23, 2021 @ 20:30 UTC) from a TestFlight beta tester:

> It would be great if all draft posts were ordered by their post date and time instead of when we last edited them. 

### Expected behavior

Have the Drafts posts ordered by post date and time rather than last edit time, or at least provide an option to choose the order

### Actual behavior

Draft posts are ordered by edit time

### Steps to reproduce the behavior

- Create multiple draft posts, with post date and time
- Edit one of the post with an earlier post date than the rest, so that its edit date jumps in front
- Refrehs the Draft Posts list page

##### Tested on [device], iOS [version], WPiOS [version]

Device: iPhone12,3
iOS Version: 14.8
App Version: 18.5 (18.5.0.3)

**Comment by designsimply** [comment]
@leandroalonso I see that you worked on the Posts List semi-recently in https://github.com/wordpress-mobile/WordPress-iOS/issues/11646, and I was wondering if you have any insight you could give us about why the order is the way it is and if you think we should consider changing it based on this user feedback?

**Comment by leandroalonso** [comment]
@designsimply this mainly mimics the behavior on the web and it's driven by the API (which returns the posts ordered this way) — and by web, I mean both `wp-admin` and Calypso.

In order to change that we'd need to add a filter on the UI and check if the API can order the posts in a different way (and if not, we'd have to implement it). Also notice that there's no way to sort out the posts in a different way on the web interfaces.

I'm afraid that cost vs benefit here is not great, and given we don't receive this feedback often I'd suggest keeping it as it is.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by jkmassel** [comment]
I agree with @leandroalonso – without much other user feedback about this I'm not eager to add a new feature just for it.

I think we could later look at allowing the user to set up whatever filters they want and retain those settings (like how WP data views work) as a catch-all for this sort of preference.
